packages/larva/lib/writeJson.js

```diff
@@ -1,46 +1,70 @@
-const path = require( 'path' );
-const chalk = require( 'chalk' );
-const fs = require( 'fs' );
+const path = require("path");
```

Comment:
There are a lot of stylistic changes here that make this diff hard to read, and make this file different from the rest of the project - the JS style should be following the linter config in the eslint package, the same as what we use in theme JS. I think the command `larva lint-fix:js /path/to/file` can be used to update the writeJson.js to the project style.

Reply:
Updated :+1: